Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.h

```diff
@@ -256,6 +262,8 @@ class MediaPlayerPrivateWebM
     static void getSupportedTypes(HashSet<String>&);
     static MediaPlayer::SupportsType supportsType(const MediaEngineSupportParameters&);
 
+    WeakPtr<WebMResourceClientParent> getWeakPtr() const { return *this; }
```

Comment:
Why do we need this?
Can we instead use `using WebMResourceClientParent::weakPtrFactory` et al instead?